pkg/querier/querier.go

```diff
@@ -18,10 +19,12 @@ import (
 
 // Config for a querier.
 type Config struct {
+	MaxLookBackPeriod time.Duration `yaml:"max_look_back_period"`
 }
 
 // RegisterFlags register flags.
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	f.DurationVar(&cfg.MaxLookBackPeriod, "querier.max-look_back_period", 0, "")
```

Comment:
Why the mixure of `-` and `_`? Also, can you add a description for the flag?

Reply:
must be a typo, sorry missed it somehow